<!--
  1 failure:  :disappointed_rel...
  0 warning: 
  1 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.3.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10065-WBFBFL2DNHFZIZIUZI4XYR7PR2HGG3SR7G7R5CQDLXFUSLWESTJA
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10065-WBFBFL2DNHFZIZIUZI4XYR7PR2HGG3SR7G7R5CQDLXFUSLWESTJA/android/cli/commands/_build.js
  3892:60  error  Unexpected template string expression  no-template-curly-in-string
  3896:1   error  Trailing spaces not allowed            no-trailing-spaces

✖ 2 problems (2 errors, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/android/cli/commands/_build.js
+++ b/android/cli/commands/_build.js
@@ -3885,8 +3885,15 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 	this.androidLibraries.forEach(libraryInfo => {
 		const libraryManifestPath = path.join(libraryInfo.explodedPath, 'AndroidManifest.xml');
 		if (fs.existsSync(libraryManifestPath)) {
+			let libraryManifestContent = fs.readFileSync(libraryManifestPath).toString();
+
+			// handle injected build variables
+			// https://developer.android.com/studio/build/manifest-build-variables
+			libraryManifestContent = libraryManifestContent.replace('${applicationId}', this.appid); // eslint-disable-line no-template-curly-in-string
+
 			const libraryManifest = new AndroidManifest();
-			libraryManifest.load(libraryManifestPath);
+			libraryManifest.parse(libraryManifestContent);
+
 			// we don't want android libraries to override the <supports-screens> or <uses-sdk> tags
 			delete libraryManifest.__attr__;
 			delete libraryManifest['supports-screens'];

